Signal/src/ViewControllers/SignalsViewController.m

```diff
@@ -549,6 +548,67 @@ - (void)presentThread:(TSThread *)thread
     });
 }
 
+- (void)presentTopLevelModalViewController:(UIViewController *)viewController
+                          animateDismissal:(BOOL)animateDismissal
+                       animatePresentation:(BOOL)animatePresentation
+{
+    OWSAssert([NSThread isMainThread]);
+    OWSAssert(viewController);
+
+    [self presentViewControllerWithBlock:^{
+        [self presentViewController:viewController animated:animatePresentation completion:nil];
+    }
+                        animateDismissal:animateDismissal];
+}
+
+- (void)pushTopLevelViewController:(UIViewController *)viewController
```

Comment:
It's not used yet since this is being added so late in the release process, but I'd like to use it to DRY up a bunch of logic inside this class.